stdlib/contrib/bonitoo-io/tickscript/tickscript.flux

```diff
@@ -145,14 +135,15 @@ compute = select
 
 // groupBy groups by specified columns.
 // It is a convenience function, it adds _measurement column which is required by monitor.check().
-groupBy = (columns, tables=<-) =>
-  tables
+groupBy = (columns, tables=<-) => tables
     |> group(columns: columns)
-    |> experimental.group(columns: ["_measurement"], mode:"extend") // required by monitor.check
+    |> experimental.group(columns: ["_measurement"], mode: "extend")
 
+// required by monitor.check
```

Comment:
For places where this change has been made, it might be tricky to undo it later. We might want to wait until we can properly format end of line comments before committing formatted stdlib files?